index.html

```diff
@@ -4,27 +4,40 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles.css">
-    <title>Welcome to my mobile portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <title>Welcome to my portfolio website</title>
 </head>
 <body>
-<div class="container">
+
+    <!-- Tollbar and Header section starts from here -->
+
+    <div class="container">
     <header class="toolbar-container">
-        <a class="welcome_logo">
-            <img src="images/Welcome logo.svg">
+        <a href="index.html">
+            <img id="logo-image" src="images/logo.svg" alt="Logo Image" />
         </a>
         <nav class="menu">
-            <img src="images/Menu.svg" alt="icon_menu">
+            <img src="images/menu.svg" alt="Menu_icon">
+        </nav>
+        <nav class="navbar">
+            <ul id="navbar-Pages">
+            <li><a href="#" class="menu-item-1">Portfolio</a></li>
+            <li><a href="#" class="menu-item-2">About</a></li>
+            <li><a href="#" class="menu-item-3">Contact</a></li>
+            </ul>
         </nav>
         </header>
         <section class="hero-title-section">
-        <h1 class="hero-title">
-    Hey there. I'm Jerome<br>
-    <span>I'm a software developer</span>
```

Comment:
- [ ] I noticed your header and hero section overlaps both desktop and mobile versions and looks different from the design template. Kindly fix this, please.
- [ ] Also add a background image in this section is not displaying correctly, kindly fix this, please. You are expected to stick to design as much as possible.
### Your design
![1](https://user-images.githubusercontent.com/94551484/194692983-e4042ffb-eb01-47e4-95b2-727b454f10a9.png)

### Figma template
![f1](https://user-images.githubusercontent.com/94551484/194693025-63a7ee38-2570-46d3-b7aa-692b37c66a53.png)